run/system-package/main.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2019 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from flask import Flask, make_response, request
+import os
+import subprocess
+import sys
+
+
+app = Flask(__name__)
+
+
+# [START run_system_package_handler]
+@app.route("/diagram.png", methods=["GET"])
+def index():
+    # Takes an HTTP GET request with query param dot and
+    # returns a png with the rendered DOT diagram in a HTTP response.
+    try:
+        image = create_diagram(request.args.get("dot"))
+        response = make_response(image)
+        response.headers.set("Content-Type", "image/png")
+        return response
+
+    except Exception as e:
+        print(f"error: {e}")
```

Comment:
This style of format string is not supported in Python < 3.6
```suggestion
        print("error: {}".format(e))
```